src/components/quickstart/Windows.astro

```diff
@@ -73,7 +77,105 @@ const { latestVersion } = Astro.props
   />
 </div>
 
-<Examples itemNumber="3/3" terminalType="powershell-ubuntu" terminalSymbol="$" />
+<div class="prose dark:prose-invert">
+  <h2 class="text-3xl mt-24">
+    <small class="block font-mono text-xs text-gray-500">{itemNumber}</small>Create a
+    Project
+  </h2>
+
+  <p>Inside the WSL2 "DDEV" distro (<strong>not on Windows</strong>), create a directory for your project or clone your project:</p>
```

Comment:
"not on Windows" - if I was Windows only user, I would not understand this.

Maybe something like:

not in `C:\` or `D:\` drive, use Explorer > `Linux` in the sidebar > `DDEV` > `/home/<you>` to see the actual files.